lib/spack/docs/containers.rst

```diff
@@ -44,8 +64,8 @@ Environments:
    &&   echo "    install_tree: /opt/software" \
    &&   echo "  view: /opt/view") > /opt/spack-environment/spack.yaml
 
-   # Install the software, remove unnecessary deps
-   RUN cd /opt/spack-environment && spack env activate . && spack install && spack gc -y
+   # Install the software, remove unecessary deps
```

Comment:
typo: "unecessary" -> "unnecessary"

Reply:
:+1: Will fix here and in the template for the Dockerfile